usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -71,11 +71,11 @@ SELECT
 
   UTM.action_date,
   DATE(UTM.action_date + interval '3 months') AS fiscal_action_date,
```

Comment:
Why not also store `fiscal_action_date` as field in the UTM matview, which is derived at matview-refresh time using the `DATE(...)` function. That would save on the expense of calculating the value at view-query-time.

Reply:
Done